README.md

```diff
@@ -1,5 +1,27 @@
 # 💬 `quotes`
 
+### Disclaimer for contributions
+
+If you are thinking of _contributing_ to this repository (hooray! :tada:),
```

Comment:
```suggestion
If you are thinking of _contributing_ to this repository (hooray! 🎉),
```

Reply:
I didn't think that adding the markdown would make a difference here! Thanks for the update!

Reply:
Some browsers / markdown parsers don't automatically display emoji, so I try to use the actual unicode character wherever possible. 👍